feature-flags/src/main/java/cm/aptoide/pt/feature_flags/domain/FeatureFlags.kt

```diff
@@ -44,4 +45,8 @@ class FeatureFlags @Inject constructor(
       result.filterNot { it.isBlank() }
     } ?: emptyList()
   }
+
+  suspend fun <T> getObject(key: String, klass: Class<T>) = mutex.withLock {
+    runCatching { Gson().fromJson(key, klass) }.getOrNull()
```

Comment:
Does it expect an actual `JSON` string provided in a `key` parameter? 🤔

Reply:
fixed